indico/modules/events/abstracts/controllers/email_templates.py

```diff
@@ -114,6 +115,22 @@ def _process_PATCH(self):
         return jsonify_data(flash=False)
 
 
+class RHQuickSetupEmailTemplates(RHManageAbstractsBase):
+    """Setups email templates notifications for every non-configured management action."""
+
+    def _process(self):
+        rules = self.event._get_email_templates_rules()
+        notificable_values = {e.value: e.name for e in AbstractNotifiedStates}
+        non_configured_notifications = {k: v for k, v in notificable_values.items() if k not in rules}
+        for k, v in non_configured_notifications.items():
+            new_tpl = build_default_email_template(self.event, v)
+            new_tpl.title = v.capitalize()
+            new_tpl.rules = [{'state': [k]}]
+            self.event.abstract_email_templates.append(new_tpl)
+        db.session.flush()
+        return redirect(url_for('abstracts.management', self.event))
```

Comment:
I think it's better to keep the form open and update it, same as we do when you e.g. delete a template. Currently you have no feedback whether the setup worked or not.

Reply:
That's precisely the reason why I decided in the end to redirect to management, because the Flash message advising you have not setup notifications will disappear (that's the feedback the user is getting)

Reply:
Why not keeping it this way and flashing another message saying it has been set up?

Reply:
I can think of two things:
- For consistency - deleting or adding a new rule doesn't close the dialog either
- As a user I'd want to double-check which notifications have been added or perhaps modify them and for that I'd need to reopen the notifications

Reply:
There's also a problem with that, and it is the RH expects a template to edit the form, but there's no form nor template, since I'm editing many of them in bulk.

Also, when tried of doing what you suggest, there was no way of refreshing the modal, even using the same RH and returning same method the edit and delete RH do to refresh it, it resulted in the opening of a new one with the fresh information, but the old one was still behind.

Reply:
This should correctly update the dialog (if you use `POST` instead of `GET` for the endpoint):

```jinja
<button class="i-button icon-plus highlight"
        data-href="{{ url_for('.quick_setup_notifications', event) }}"
        data-title="{% trans %}Add a new notification ruleset{% endtrans %}"
        data-update="#email-template-manager"
        data-method="POST">
    {% trans %}Quick setup{% endtrans %}
</button>
```

Notice that I'm using `data-method=...` instead of `data-ajax-dialog`